File: backend/src/main/java/org/jdk/project/dto/digitaltwin/DigitalTwinInspectionDto.java
package org.jdk.project.dto.digitaltwin;

import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
public class DigitalTwinInspectionDto {
  private Long id;
  private Long inspTaskId;
  private String robotId;
  private String areaName;
  private String roomCode;
  private String abnormal;
  private String abnormalInfo;
  private String coordinate;
}

File: backend/src/main/kotlin/org/jdk/project/dto/digitaltwin/DigitalTwinInspectionDto.kt
package org.jdk.project.dto.digitaltwin

data class DigitalTwinInspectionDto(
    var id: Long? = null,
    var inspTaskId: Long? = null,
    var robotId: String? = null,
    var areaName: String? = null,
    var roomCode: String? = null,
    var abnormal: String? = null,
    var abnormalInfo: String? = null,
    var coordinate: String? = null,
)
